### DISCUSSION TITLE
How to create and fill up the config.ini in Pythonanywhere?

### DISCUSSION THREAD

**Discussion by sarmad84** [aahnik/tgcf#48] [Q&A] [answered]
Could you please explain how to create and fill up the config.ini file with my forwarding configurations in Pythonanywhere?

**Comment by aahnik** [comment, marked as answer]
See #23 , python anywhere has a built in file explorer and text editor.

You can also edit from terminal using stuff like nano.

Please google search and play around with python anywhere. Don't just ask anything that comes to your mind.

**Comment by sarmad84** [comment]
I get this error when I run it:

![image](https://user-images.githubusercontent.com/25746389/113499233-91752900-9557-11eb-9f18-e702820121e1.png)

**Reply by aahnik** [reply]
https://github.com/aahnik/tgcf/discussions/23#discussioncomment-410020

already discussed.
